src/ProjectWindow.cpp

```diff
@@ -595,45 +596,35 @@ ProjectWindow::ProjectWindow(wxWindow * parent, wxWindowID id,
    mTopPanel->SetBackgroundColour(theTheme.Colour( clrMedium ));
 #endif
 
-   wxWindow    * pPage;
+   auto container = safenew wxSplitterWindow(this, wxID_ANY,
+      wxDefaultPosition,
+      wxDefaultSize,
+      wxNO_BORDER | wxSP_LIVE_UPDATE);
+   container->Bind(wxEVT_SPLITTER_DOUBLECLICKED, [](wxSplitterEvent& event){
+      //"The default behaviour is to unsplit the window"
+      event.Veto();//do noting instead
+   });
+   container->SetBackgroundColour(theTheme.Colour(clrLight));
```

Comment:
What happens if the user changes the theme?  Do you need to add to `ProjectWindow::OnThemeChange` to reset this color?  Or, is the background color in fact not seen because everything else in the window hides it?

Reply:
What happens if you try to drag the separator?  Is that not wanted, and do you need to Bind and Veto another event?

Reply:
>What happens if the user changes the theme? Do you need to add to ProjectWindow::OnThemeChange to reset this color? Or, is the background color in fact not seen because everything else in the window hides it?

In my prototype branch I had changed background color in response to the theme event, but there was no visible effect, because of the child windows. I mistakenly committed this line.

>What happens if you try to drag the separator? Is that not wanted, and do you need to Bind and Veto another event?

If I understand you correctly you saying that we don't need the separator to be draggable?

Reply:
I ask whether it is intended.

maybe a limit on rightward dragging is needed.